static/src/javascripts/projects/common/modules/commercial/creatives/add-tracking-pixel.js

```diff
@@ -0,0 +1,17 @@
+define([
+    'common/utils/template',
+    'text!common/views/commercial/tracking-pixel.html'
+], function (
+    template,
+    trackingPixelStr
+) {
+    var trackingPixelTpl;
```

Comment:
this is temporary anyway, I have my precompiled templates nearly ready for take out :smile: